lib/dotcom_web/controllers/mode/hub_behavior.ex

```diff
@@ -1,7 +1,28 @@
 defmodule DotcomWeb.Mode.HubBehavior do
   @moduledoc "Behavior for mode hub pages."
 
-  use DotcomWeb, :controller
+  use Dotcom.Gettext.Sigils
```

Comment:
📓 I just moved what was in `DotcomWeb, :controller` into this module because it broke when trying to add the decorator.

Reply:
The mode hub pages are `/schedules/(bus|ferry|commuter-rail|subway)`, each using a separate controller which implements this behavior!